shared/login/register/set-public-name/container.js

```diff
@@ -84,14 +83,14 @@ type OwnProps = RouteProps<
 >
 
 const mapStateToProps = (state: TypedState, {routeProps}: OwnProps) => ({
-  deviceNameError: state.signup.deviceNameError,
+  deviceNameError: state.signup.devicenameError,
   existingDevices: routeProps.get('existingDevices'),
   existingDevicesTrimmed: trimDeviceNames(routeProps.get('existingDevices')),
   waiting: state.engine.get('rpcWaitingStates').get('loginRpc'),
 })
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
-  clearDeviceNameError: () => dispatch(SignupGen.createClearDeviceNameError()),
+  clearDeviceNameError: () => {},
```

Comment:
is this still needed?

Reply:
nope. i'm going to fix this in part 5 so i can validate its all working: https://github.com/keybase/client/pull/12551/commits/218f1b3556ab94b5261042b28196ae83dff4f77a